src/pages/GetHelp.vue

```diff
@@ -94,18 +98,18 @@
                     >
                       <div class="q-gutter-md row">
                         <q-date
-                          mask="DD/MM/YYYY HH:mm"
+                          mask="YYYY-MM-DDTHH:mm:ss"
                           v-model="enddate"
                           color="secondary"
-                          :rules="['date']"
+                          :rules="['datetime']"
                           :options="fromToday"
                         />
                         <q-time
                           v-model="enddate"
-                          mask="DD/MM/YYYY HH:mm"
-                          :rules="['time']"
+                          mask="YYYY-MM-DDTHH:mm:ss"
+                          :rules="['datetime']"
                           color="secondary"
-                          @input="() => $refs.endDateProxy.hide()"
+                          @input="$refs.endDateProxy.hide()"
```

Comment:
die Funktioun auch bitte nicht invoken
Einfach die Referenz zur Funktion geben
@input="$refs.endDateProxy.hide"

Reply:
Hmm if I do `@input="$refs.endDateProxy.hide"` the Date is not popping up.

Reply:
not popping up, you mean not hiding ?

Reply:
let's kee it then for now, we'll figure it out later

Reply:
Nope really, not popping up